packages/mobile/src/escrow/EscrowedPaymentListItem.tsx

```diff
@@ -21,41 +21,45 @@ interface OwnProps {
 
 type Props = OwnProps & WithNamespaces
 
-export class EscrowedPaymentReminderNotification extends React.PureComponent<Props> {
-  getCTA = () => {
+export class EscrowedPaymentListItem extends React.PureComponent<Props> {
+  onSendMessage = () => {
     const { payment, t } = this.props
     const recipientPhoneNumber = payment.recipientPhone
+    CeloAnalytics.track(CustomEventNames.clicked_escrowed_payment_send_message)
+    // TODO: open up whatsapp/text message slider with pre populated message
+    try {
+      if (Platform.OS === 'android') {
+        SendIntentAndroid.sendSms(recipientPhoneNumber, t('escrowedPaymentReminderSms'))
+      } else {
+        // TODO look into using MFMessageComposeViewController to prefill the body for iOS
+        navigateToURI(`sms:${recipientPhoneNumber}`)
+      }
+    } catch {
+      Logger.showError(t('SMSError'))
```

Comment:
We don't provide strings directly to show error. You should create an entry in the ErrorMessages enum instead